composer.json

```diff
@@ -29,10 +29,10 @@
     "require-dev": {
         "roave/security-advisories": "dev-latest",
         "codeception/codeception": "^5.0",
-        "squizlabs/php_codesniffer": "^3.6.0",
+        "squizlabs/php_codesniffer": "^3.6",
         "overtrue/phplint": "^4.3",
         "phpunit/phpunit": "^9.5",
-        "codeception/phpunit-wrapper": "^9.0.6"
+        "codeception/phpunit-wrapper": "^9.0"
```

Comment:
With Codeception 5, phpunit-wrapper is no longer necessary.
symfony/finder constraint should match Codeception 5: `"symfony/finder": ">=4.4.24 <7.0",`

Reply:
Thanks a lot. I will change it